### PR TITLE
chore(playground): add new options

### LINKED ISSUE #3338: Support new options on playground
- `arrowParens`
- `proseWrap: "preserve"`

### PR DESCRIPTION
**Pull request by ikatyang**
Fixes #3338

- [x] wait for #3340

### CODE REVIEW

File: src/cli-constant.js [hunk 1]
```diff
@@ -82,7 +82,6 @@ const detailedOptions = normalizeDetailedOptions({
     category: CATEGORY_FORMAT,
     forwardToApi: true,
     description: "Include parentheses around a sole arrow function parameter.",
-    default: "avoid",
```
Comment (suchipi) [inline src/cli-constant.js]: Are you going to fill this back in once we've decided on the new default?

Comment (ikatyang) [reply src/cli-constant.js]: Oops, forgot to mention why I removed this line: we use the default value from `options.js` for all format options.

https://github.com/prettier/prettier/blob/fbbfa525171f36632ae879010cba92af9a5fd896/src/cli-util.js#L561-L579

Comment (suchipi) [reply src/cli-constant.js]: Hmm, that's feels a bit unintuitive, but changing it is out of the scope of this PR

**Comment by azz** [comment]
Currently the playground uses the released version of Prettier (from package.json) so this won't actually work if we merge it.

**Comment by duailibe** [comment]
I added the 1.9 milestone just to help us remember it's related but feel free to remove if you think someone might think this is a blocker.

**Comment by suchipi** [comment]
So to clarify, this should be merged post release?

**Comment by azz** [comment]
Yes. After prettier's own dependency on prettier is updated to 1.9.0

**Pull request merged**